### ISSUE TITLE
Unable to open https://screenshottocode.com/ 

### ISSUE THREAD

**Issue by smartVaibhaviG** [abi/screenshot-to-code#451]
**Describe the bug**
**https://screenshottocode.com/** 
I am not able to open this URL from a couple of days. Getting loader only.

**To Reproduce**
1. Go to browser
2. Hit "https://screenshottocode.com/ " this URL
3. check

**Comment by abi** [comment]
@smartVaibhaviG thanks for the report. Can you screenshot the developer console for me in the browser (Ctrl  or Cmd + Shift + J)? Could you try in a different browser?

**Comment by smartVaibhaviG** [comment]
Yes, the same issue is also being faced in another browser. I have shared screenshots for reference.

![Screenshot from 2024-12-06 15-16-58](https://github.com/user-attachments/assets/4c1fc1f1-0df6-46e7-8461-8b75cc531f9e)
![Screenshot from 2024-12-06 15-17-37](https://github.com/user-attachments/assets/18163c22-2506-480a-8b48-cfe435232467)

**Comment by abi** [comment]
I think it's clerk.screenshottocode.com (our auth provider) is being blocked on your Internet. If you have a VPN, using that should definitely fix it. If not, would you happen to have any anti virus software installed? If so, please let me know and I can get the website removed from any blacklists.
